package.json

```diff
@@ -18,9 +18,8 @@
     "gulp": "^3.9.1",
     "nodemon": "^1.8.1",
     "origami-build-tools": "^5.0.0",
-    "whatwg-fetch": "^0.10.1"
-  },
-  "dependencies": {
-    "ftlabs-screens-viewer": "^3.0.0"
+    "whatwg-fetch": "^0.10.1",
+	"ftlabs-screens-carousel": "github:ftlabs/screens-carousel#v1.0.8",
```

Comment:
Is it wise to lock this off to a specific version? Could we not take the same approach as we have with the screens-viewer module and build from npm? That way, we can use semver.